src/main/java/io/cryostat/net/web/http/api/v2/TargetCredentialsPostHandler.java

```diff
@@ -105,17 +104,19 @@ public boolean isOrdered() {
     @Override
     public IntermediateResponse<Void> handle(RequestParameters params) throws ApiException {
         String targetId = params.getPathParams().get("targetId");
-        String username;
-        String password;
-        try {
-            username =
-                    Objects.requireNonNull(
-                            params.getFormAttributes().get("username"), "Username is required");
-            password =
-                    Objects.requireNonNull(
-                            params.getFormAttributes().get("password"), "Password is required");
-        } catch (NullPointerException npe) {
-            throw new ApiException(400, npe.getMessage(), npe);
+        String username = params.getFormAttributes().get("username");
+        String password = params.getFormAttributes().get("password");
+
+        if (username == null || password == null) {
+            StringBuilder sb = new StringBuilder();
+            if (username == null) {
+                sb.append("username is required.");
+            }
+            if (password == null) {
+                sb.append(" password is required.");
```

Comment:
I think username/password should be capitalized.

Reply:
I left them lowercased here since that reflects the form attribute name, although perhaps I should make that more explicit in the message?

Reply:
Although, form attributes are actually case-insensitive - I suppose it doesn't really matter in that case. I can capitalize these messages if you do prefer.

Reply:
Okay, scratch that - from what I can determine, the spec doesn't say if form parameters are case insensitive, so they should probably be assumed to be case-sensitive.

In that case my preference would be to leave the attribute names lower case in the error message, but to include a bit more detail in the message to indicate that we're talking about a form attribute of the given name being required. What do you think?

Reply:
Hmm I thought capitalized would be better since there are periods, but that's a good point. In that case maybe the attributes should be quoted? It looks like that's how it's done for the RulesPostHandler:
```
{"meta":{"type":"text/plain","status":"Bad Request"},"data":{"reason":"\"targetAlias\" cannot be blank, was \"null\""}}
```

Reply:
Just saw your last comment. Yes I think lower case makes sense, and probably with quotes it should be obvious enough.